levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 1234567 need host pin - [18/Feb/2019 17:04:31]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 1234567 has incorrect number range - [18/Feb/2019 17:05:40]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 1234567 has incorrect number range - [18/Feb/2019 17:10:45]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr %s should have different guest and host passcode - [18/Feb/2019 17:11:10]
levelname:INFO filename: vmr(host_guest)_add_tool.py outputNumber: [109]  func: add_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 17:15:43]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 392609111 already exist - [18/Feb/2019 17:16:05]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 12345 has incorrect number range - [18/Feb/2019 17:18:43]
levelname:ERROR filename: vmr_del_tool.py outputNumber: [79]  func: <module> output msg:  vmr 392609111 doesnt exist or has multi ids - [18/Feb/2019 17:20:41]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 392609111 has incorrect host pin value - [18/Feb/2019 17:23:21]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 392609111 need host pin - [18/Feb/2019 17:24:08]
levelname:INFO filename: vmr(host_guest)_add_tool.py outputNumber: [109]  func: add_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 17:24:27]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr %s should have different guest and host passcode - [18/Feb/2019 17:24:48]
levelname:ERROR filename: vmr(host_guest)_add_tool.py outputNumber: [131]  func: <module> output msg:  vmr 392609111 already exist - [18/Feb/2019 17:25:13]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  vmr 39260911 is in incorrect number range - [18/Feb/2019 17:47:54]
levelname:INFO filename: vmr_modify_tool.py outputNumber: [101]  func: modify_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 17:48:35]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  exceptions must derive from BaseException - [18/Feb/2019 17:51:09]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  exceptions must derive from BaseException - [18/Feb/2019 17:53:28]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  vmr 392609111 has incorrect calllegprofile value - [18/Feb/2019 17:54:02]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  vmr 392609111 should have different guest and host passcode - [18/Feb/2019 17:54:41]
levelname:INFO filename: vmr_modify_tool.py outputNumber: [101]  func: modify_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 17:55:51]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 17:56:20]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 18:01:11]
levelname:ERROR filename: vmr_retrieve_tool.py outputNumber: [151]  func: <module> output msg:  [Errno 13] Permission denied: './VMR_list.xlsx' - [18/Feb/2019 18:01:11]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 18:01:17]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 18:02:06]
levelname:ERROR filename: vmr_modify_tool.py outputNumber: [120]  func: <module> output msg:  vmr 392609111 host has incorrect calllegprofile value - [18/Feb/2019 18:02:23]
levelname:INFO filename: vmr_modify_tool.py outputNumber: [101]  func: modify_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 18:03:05]
levelname:INFO filename: vmr_modify_tool.py outputNumber: [101]  func: modify_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 18:03:20]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 18:03:26]
levelname:INFO filename: vmr_modify_tool.py outputNumber: [101]  func: modify_vmrs output msg:  vmr 392609111 is done - [18/Feb/2019 18:04:49]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [18/Feb/2019 18:04:54]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [20/Feb/2019 17:55:58]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [20/Feb/2019 17:56:30]
levelname:INFO filename: vmr_retrieve_tool.py outputNumber: [124]  func: write_values_to_excel output msg:  vmr 392609111 is done - [20/Feb/2019 17:57:05]
